### ISSUE TITLE
4.2.1 Broke the local dev server

### ISSUE THREAD

**Issue by timothystewart6** [jaredpalmer/razzle#1733]
### Current Behavior
`yarn start`

```
./node_modules/razzle-dev-utils/webpackHotDevClient.js
Module not found: Can't resolve 'webpack-dev-server/client/utils/createSocketURL' in '/app/node_modules/razzle-dev-utils'
```

after upgrading to `4.2.1` it seems like the local dev server no longer works

```
❯◯ razzle            ^4.0.6  4.1.0  ❯  4.2.1  https://github.com/jaredpalmer/razzle#readme
 ◯ razzle-dev-utils  ^4.0.6  4.1.0  ❯  4.2.1  https://github.com/jaredpalmer/razzle#readme
```

### Expected behavior

dev server to work

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | 4.2.1
| Razzle Plugins   |
| Node             | 14.17.4
| Browser          |
| npm/Yarn         |
| Operating System |
| TypeScript       |
| React            |

**Comment by fivethreeo** [comment]
do a yarn why webpack-dev-server

**Comment by timothystewart6** [comment]
```
=> Found "webpack-dev-server@3.11.2"
info Has been hoisted to "webpack-dev-server"
info This module exists because it's specified in "devDependencies".
info Disk size without dependencies: "1.64MB"
info Disk size with unique dependencies: "7.15MB"
info Disk size with transitive dependencies: "32.96MB"
info Number of shared dependencies: 176
Done in 1.01s.
```

**Comment by timothystewart6** [comment]
Related to https://github.com/jaredpalmer/razzle/pull/1730?  I figured it would still be compatible with 3

**Comment by fivethreeo** [comment]
It is meant to be compatible with both. 

https://github.com/jaredpalmer/razzle/commit/a56bb34b715f5bde48acb21a9c2f08d52c9096f3

https://github.com/jaredpalmer/razzle/pull/1730#issuecomment-944851925

**Comment by timothystewart6** [comment]
Hmmm... well it doesn't seem to be with the combination I am using.

**Comment by fivethreeo** [comment]
I would switch back to 4.2.0 for the time being. Need to investigate this. Multi version deps is always trouble.

**Comment by timothystewart6** [comment]
Ok no worries.  Happy to help and test.  Noticed both files exist in my node modules.

**Comment by fivethreeo** [comment]
May be a issue with webpack-dev-server then. I will revert and release 4.2.2.

**Comment by fivethreeo** [comment]
released 4.2.2 with reverted module path fix, issue with filename seems to be a webpack-dev-server issue

**Comment by timothystewart6** [comment]
Thanks, that fixed it.
